core/include/kernel/transfer_list.h

```diff
@@ -22,7 +22,22 @@
  * Version of the register convention used.
  * Set to 1 for both AArch64 and AArch32 according to fw handoff spec v0.9
  */
-#define REG_CONVENTION_VER_MASK BIT(24)
+#define REG_CONVETIONN_VER_SHIFT_64	UL(32)
+#define REG_CONVETIONN_VER_SHIFT_32	UL(24)
+#define REG_CONVENTION_VER_MASK	UL(0xff)
+#define REG_CONVENTION_VER	UL(1)
+
+#define TRANSFER_LIST_HANDOFF_X1_VALUE(__version)	\
```

Comment:
Thanks :) However, when `TL REG_CONVENTION_VER` could be changed,
I think it would be better to rename 
`TRANSFER_LIST_HANDOFF_X1_VALUE` to `TL_HANDOFF_X1_VALUE`

Reply:
Sure, that helps too, perhaps that's short enough to handle some indentation. I guess we'll see how it turns out in `boot_save_args()`.